packages/grid_client/src/modules/models.ts

```diff
@@ -572,6 +572,7 @@ class CalculatorModel {
   @Expose() @IsNumber() @IsNotEmpty() @Min(0) hru: number; // GB
   @Expose() @IsBoolean() @IsNotEmpty() ipv4u: boolean;
   @Expose() @IsBoolean() @IsOptional() certified?: boolean;
+  @Expose() @IsBoolean() @IsOptional() dedicated?: boolean;
```

Comment:
what is that field used for?

Reply:
I removed it as there was no need for it.